.github/checks/safety.sh

```diff
@@ -138,6 +138,10 @@ while getopts hamuiprebldxcft opt; do
         p)
             for file in $files
             do
+                if [[ $file == './tremor-connector-derive/'* ]]; then
```

Comment:
we have an .checkignore file that could house this instead of adjusting the script :)

Reply:
AFAIK `.checkignore` stops all the checks from happening, but I think for the derives, all the others make sense, it just needs to be able to panic.

Reply:
That makes sense, would a `// ALLOW: this is a derive macro so panicking is the way to error` in the derive be an alternative? Hardcoding the file path in the check script seems wrong to me o.O especially when we have alternative ways to deal with it already.